pkg/security/module/server.go

```diff
@@ -234,7 +237,7 @@ func (a *APIServer) dequeue(now time.Time, cb func(msg *pendingMsg) bool) {
 			return true
 		}
 
-		if msg.retry >= maxRetry {
+		if msg.skip || msg.retry >= maxRetry {
```

Comment:
Shouldn't we having logging an error when the event is dropped because of filters?
Also, what do you think about adding a metric for the number of events that were dropped because of this?